types/n3/index.d.ts

```diff
@@ -130,6 +130,24 @@ export interface BlankTriple<Q extends RDF.BaseQuad = RDF.Quad> {
     object: Q['object'];
 }
 
+export interface Token {
+    type: string;
+    value?: string;
+    line: number;
+}
+export interface LexerOptions {
+    lineMode?: boolean;
+    n3?: boolean;
+    comments?: boolean;
+}
+
+export type TokenCallback = (error: Error, token: Token) => void;
+
+export class Lexer {
+    constructor(options?: LexerOptions);
+    tokenize(input: string | EventEmitter): Token[];
```

Comment:
```suggestion
    tokenize(input: string): Token[];
```
A callback seems to be required when the input is an EventEmitter: https://github.com/rdfjs/N3.js/blob/89f0daf748bf2d6ca194b37e58733798346a88e0/src/N3Lexer.js#L455-L485